atlas/packages/app/src/components/Navbar/Navbar.tsx

```diff
@@ -37,7 +37,7 @@ const Navbar: React.FC<RouteComponentProps> = () => {
           <Link to="/">
             <StyledIcon name="home" />
           </Link>
-          <Link to="/discover">
+          <Link to="/browse">
```

Comment:
We should use the value from `routes` object here

Reply:
Fixed, thanks!